### ISSUE TITLE
MultiSelect messing up accessibility due to ARIA role?

### ISSUE THREAD

**Issue by csjh** [carbon-design-system/carbon-components-svelte#1071]
Hello,

I'm making a web app with this library and it's going great, but I've run into some trouble with the accessibility of the MultiSelect component. Lighthouse audits throw the complaint `Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children.`. Looking into it, it appears the MultiSelect has the role `listbox` but doesn't have any children with the role `option`. 

Is there a fix I'm missing with a prop or something of the like, or is this a bug? I can't look into it right now, but I suspect changing [this line of MultiSelect](https://github.com/carbon-design-system/carbon-components-svelte/blob/cbecfd41e254e66d85fcf38814dc4a53fb68709e/src/MultiSelect/MultiSelect.svelte#L297) to `role="option"` might fix? option is the correct role for a child of listbox [according to mdn](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role).

Thanks,
csjh
